src/main/java/com/selfxdsd/todocli/JsonTodosSerializer.java

```diff
@@ -87,12 +87,7 @@ public URI serialize() {
     private File getFile(final boolean deleteFirst) throws URISyntaxException,
             IOException {
         //folder where application is running.
-        final String parent = new File(JsonTodosSerializer.class
-                .getProtectionDomain()
-                .getCodeSource()
-                .getLocation()
-                .toURI())
-                .getParent();
+        final File parent = new File(System.getProperty("user.dir"));
```

Comment:
@criske I understand ``user.dir`` is predefined by the JDK and always points to the root dir where the app is running?

Reply:
@amihaiemil As far I was remembering, is the folder where `java` command is called.
But anyway, just to be sure, I've tested on linux too. In `self-web` I ran `java -jar ~/todo-cli/todo-finder-cli-0.0.5-SNAPSHOT.jar` and `todos.json` was generated in `self-web` folder.